tests/integration/introduction/test_introduction.py

```diff
@@ -42,3 +43,33 @@ def test_intro_basis_for_completion_not_displayed(self):
         # When on the introduction page
         # Then basis for completion should not be displayed
         self.assertNotInPage('basis-for-completion')
+
+    def test_start_survey_sets_started_at(self):
+        # Given survey with a start survey button
+        self.launchSurvey('1', '0112', roles=['dumper'])
+
+        self.post(action='start_questionnaire')
+
+        # When start survey button is pressed,
+        # Then started_at should be set in collection_metadata (and payload)
+        actual = self.dumpSubmission()['submission']
+
+        self.assertIsNotNone(actual['started_at'])
```

Comment:
Shouldn't this check if the value of started at is a valid date format , it could be an empty string ?

Reply:
Yep, I'll sort that out.